contract/contracts/Voting/Voting.sol

```diff
@@ -37,18 +52,17 @@ contract Voting {
 
```

Comment:
i just saw this code
```solidity
for (uint256 i = 0; i < _proposal.candidates.length; i++) {
    newProposal.candidates.push(_proposal.candidates[i]);
}

for (uint256 i = 0; i < _proposal.voters.length; i++) {
    newProposal.voters.push(_proposal.voters[i]);
}
```

in solidity we can't do something like this? to avoid looping
```solidity
newProposal.candidates = _proposal.candidates;
newProposal.voters = _proposal.voters
```

Reply:
Copying of type struct Types.Candidate calldata[] calldata to storage not yet supported.